spyder/plugins/debugger/utils/breakpointsmanager.py

```diff
@@ -10,16 +10,20 @@
 import os.path as osp
 
 from qtpy.QtWidgets import QInputDialog, QLineEdit
+from qtpy.QtCore import Signal, QObject
 
 from spyder.config.manager import CONF
 from spyder.config.base import _
 from spyder.py3compat import to_text_string
 from spyder.api.manager import Manager
 from spyder.plugins.editor.utils.editor import BlockUserData
+from spyder.plugins.debugger.panels.debuggerpanel import DebuggerPanel
+from spyder.plugins.editor.utils.editor import get_file_language
+from spyder.plugins.editor.utils.languages import ALL_LANGUAGES
 
 
 def _load_all_breakpoints():
```

Comment:
I am not so sure because now the CONF is updated once instead of once for every editor. If I add these methods to `BreakpointsManager` I would loose this. Also what if the editor plugin is closed?

Reply:
> I am not so sure because now the CONF is updated once instead of once for every editor. If I add these methods to BreakpointsManager I would loose this

Ok, I see. I thought my suggestion could be implemented easily, but we can see how to do it later.

> Also what if the editor plugin is closed?

What do you mean by this?

Reply:
If there is no open codeeditor, but I request to remove all breakpoints, there are no `BreakpointsManager` to remove all breakpoints.